database/postgres/vars.tf

```diff
@@ -37,4 +37,9 @@ variable "engine_version" {
   type        = string
   description = "RDS engine version"
   default     = null
-}
\ No newline at end of file
+}
+variable "db_instance_class" {
+  type        = string
+  description = "RDS database instance class"
+  default     = null
```

Comment:
This breaks the module, compared to previous usage. If you checkout the _sub/database/postgres submodule you can see that the default is "db.t2.micro", so your default should also reflect this if you don't want to break stuff.

Reply:
Fixed, default now the same as in the submodule.